bids/analysis/transformations/compute.py

```diff
@@ -49,11 +54,18 @@ def _transform(self, var, model='spm', derivative=False, dispersion=False,
         elif model != 'fir':
             raise ValueError("Model must be one of 'spm', 'glover', or 'fir'.")
 
-        convolved = hrf.compute_regressor(vals, model, onsets,
-                                          fir_delays=fir_delays, min_onset=0)
+        min_interval = min(np.ediff1d(np.sort(var.onset)).min(),
+                           var.duration.min())
+        oversampling = np.ceil(1 / (min_interval * sampling_rate))
```

Comment:
We should probably add a test to make sure that `oversampling` is computing properly from the input parameters (which might require mocking `compute_regressor`).